src/Moneybird.Net/Models/SalesInvoices/SalesInvoiceSendOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Moneybird.Net.Misc;
+using System.Text.Json.Serialization;
+
+namespace Moneybird.Net.Models.SalesInvoices
+{
+    public class SalesInvoiceSendOptions
+    {
+        [JsonPropertyName("delivery_method")]
+        public DeliveryMethod DeliveryMethod { get; set; }
```

Comment:
Props for re-using this enum but it seems like the enum has a value 'Post' (created to use for contact entities) which is not accepted by the send invoice endpoint. Did you think about this?

Reply:
I see, let's park this issue for now. Thanks for the elaborate explanation!